src/components/bulk-email-tool/BulkEmailBody.jsx

```diff
@@ -1,5 +1,25 @@
-import React from 'react';
+import React, { useRef } from 'react';
+import { Button, Form } from '@edx/paragon';
+
+import TextEditor from './TextEditor';
 
 export default function BulkEmailBody() {
-  return <div />;
+  const editorRef = useRef(null);
+  const onInit = (event, editor) => { editorRef.current = editor; };
+
+  return (
+    <div className="w-100 m-auto p-lg-4 p-2.5">
+      <Form>
+        <Form.Group controlId="emailSubject">
+          <Form.Label>Subject:</Form.Label>
+          <Form.Control className="w-lg-50" />
+        </Form.Group>
+        <Form.Group controlId="emailBody">
+          <Form.Label>Body:</Form.Label>
+          <TextEditor onInit={onInit} />
+        </Form.Group>
+        <Button variant="primary" type="submit">Submit</Button>
```

Comment:
Should we be doing the i18n work now? Otherwise we will need to come back later and re-do this, right? This goes for line 14 (`subject`) and line 18 (`Body`) as well.

Reply:
Good point. Working on it now 🔧